docs/en/integrations/data-ingestion/kafka/kafka-clickhouse-connect-sink.md

```diff
@@ -85,18 +85,29 @@ To connect the ClickHouse Sink to the ClickHouse server, you need to provide:
 
```

Comment:
The comment above also must be adjusted to align with the list of `Required` options